web/source/kmwembedded.ts

```diff
@@ -351,12 +351,20 @@ namespace com.keyman.text {
     }
   };
 
+  /**
+   * Register and enable lexical model
+   */
+  keymanweb['registerModel']=function(model: com.keyman.text.prediction.ModelSpec) {
+    keymanweb.modelManager.register(model);
+    keymanweb.modelManager.enabled = true;
```

Comment:
Please eliminate this line, at least when we reach production.  That should be managed automatically through already-existing control structures.